spec/javascripts/app/pages/contacts_spec.js

```diff
@@ -297,7 +297,7 @@ describe("app.pages.Contacts", function(){
       expect(app.views.ConversationsForm.prototype.initialize).toHaveBeenCalled();
 
       var prefill = app.views.ConversationsForm.prototype.initialize.calls.mostRecent().args[0].prefill;
-      var contacts = app.contacts.filter(function(contact) { return contact.inAspect(app.aspect.get("id")); });
+      var contacts = app.contacts.filter(function(contact) { return contact.person.get("relationship") === "mutual" && contact.inAspect(app.aspect.get("id")); });
```

Comment:
Line 300 exceeds the maximum line length of 120.

Reply:
@denschub thanks for fixing the bot :cookie: